docs/en/ingest-management/commands.asciidoc

```diff
@@ -25,6 +26,33 @@ run these commands.
 <hr>
 ++++
 
+[discrete]
+[[elastic-agent-status-command]]
+== elastic-agent status
+
+Returns the current status of the running Elastic Agent daemon.
+
+[discrete]
+=== Synopsis
+
+[source,shell]
+----
+elastic-agent status [--output <string>]
+                     [--help]
+                     [global-flags]
+----
+
+[discrete]
+=== Options
+
+`--output <string>`::
+Output the status information in either human, json, or yaml. (default "human")
+
+`--help`::
+Show help for the `status` command.
+
+{global-flags-link}
+
```

Comment:
Can you add an example like we show in other sections? It just removes any ambiguity about the correct syntax. In this case, it might be nice to show what the command returns (unless that's likely to change a lot over time).